.env.example

```
@@ -11,7 +11,7 @@ OPENAI_API_KEY=sk-proj-your-openai-key             # 必需 (使用 Codex 时)
 
 # AI 提供者配置 (可选，都有默认值)
 AI_DEFAULT_PROVIDER=claude                         # 可选，默认: claude
-CLAUDE_DEFAULT_MODEL=claude-sonnet-4-20250514      # 可选，默认: claude-sonnet-4-20250514
+CLAUDE_DEFAULT_MODEL=claude-sonnet-4-5-20250929    # 可选，默认: claude-sonnet-4-20250514
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)